test/server/cards/02_SHD/units/RecklessGunslinger.spec.ts

```diff
@@ -0,0 +1,22 @@
+describe('Reckless Gunslinger', function () {
+    integration(function (contextRef) {
+        describe('Reckless Gunslinger\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['reckless-gunslinger']
+                    },
+                    player2: {}
```

Comment:
nit: can remove the player2 object completely if it's empty

Reply:
@AMMayberry1 - good pickup, it has been removed.